src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const priceADay = 40;
+  const LONG_TERM = 7;
```

Comment:
const name should be write in camel case
Rechange everywhere

Reply:
I don't see the mistakes